CODE_STYLEGUIDE.md

```diff
@@ -0,0 +1,174 @@
+
+## SCSS
+
+### Syntax & Formatting
+
+A just-the-facts guide to properly written multi-line CSS rules:
+
+- Use one discrete selector per line in multi-selector rulesets
+- Opening brace on the same line as the rule set
+- Include a single space before the opening brace of a ruleset
+- Properties are grouped by type: box, border, background, text, other
+- Include one declaration per line in a declaration block
+- Two (2) spaces before each declaration (no tabs)
+- Include a single space after the colon of a declaration
+- Include a semi-colon at the end of the last declaration in a declaration block
+- Place the closing brace of a ruleset in the same column as the first character of the ruleset
+- Make meaningful use of whitespace to separate rulesets and comments
+- Prefer HSL color values to RGB to longform hex to shortform hex to color names
+- Prefer single quoted strings
+- Quote attribute values in selectors
+- Where allowed, avoid specifying units for zero-values
+- Re: Zeroes, always display leading zeroes and never display trailing zeroes
+- Include a space after commas
+- Local variables should be declared before property declarations
+- Mixin inclusions with no `@content` come before property declarations
+
+```scss
+.planet,
+.asteroid {
+  display: block;
+  padding: 0.5em;
+  border: 0;
+  color: hsl(240, 6%, 12%);
+
+  &:hover {
+    background-image: url('/images/orbit.svg');
+  }
+}
+```
+
+#### Strings
+
+##### Encoding
+
+To avoid any issues with character encoding, be sure to declare UTF-8 encoding via `@charset`.
+
+```scss
+@charset 'utf-8';
+```
+
+##### Quotes
+
+While CSS and Sass don't require strings to be quoted, it's good practice for a few reasons:
+
+- Quoting helps differentiate strings from identifiers like `initial` or `serif`.
+- Color names are parsed as identifiers when unquoted
+- Syntax highlighters often fail when encountering unquoted strings
+
+If a string contains single quotes (`'`), you may opt to wrap the string in double quotes (`"`) instead of escaping (`\'`).
+
+#### Numbers
+
+##### Calculations
+
+Arithmetic should always be wrapped in parentheses to improve readability and ensure the contents are evaluated appropriately. This is true for both Sass math and extended `calc()`.
+
+```scss
+width: (100% / 3);
+
+width: calc((100% / 3) - 2em);
+```
+
+##### Magic Numbers
+
+Otherwise known as an "unnamed numerical constant", magic numbers are fixed values without a logical derivation.
+
+**They should be avoided at all costs.** This <a href="https://css-tricks.com/magic-numbers-in-css/">CSS Tricks article</a> does a great job of illustrating the possible pitfalls.
+
+If no other solution can be found, please include a comment to inform the next contributor how you arrived at the existing value.
+
+#### Colors
+
+Color literals should not be used in property delcarations. Instead, use a known color variable or design token.
+
+```scss
+color: cv('red', '500');
+
+color: $text-body;
+```
```

Comment:
```suggestion
Color literals should not be used in property declarations. Instead, use a known design token to ensure proper theming and forward compatibility.

```scss
// Don't
color: #ff0000;
color: red;

// Do
color: $text-body;
```
```

Small change here from our last pass on this.